lnbits/extensions/admin/templates/admin/index.html

```diff
@@ -0,0 +1,503 @@
+{% extends "base.html" %} {% from "macros.jinja" import window_vars with context
+%} {% block page %}
+<div class="row q-col-gutter-md justify-center">
+  <div class="col q-my-md">
+    <q-btn
+      label="Save"
+      color="primary"
+      @click="updateSettings"
+      :disabled="!checkChanges"
+    >
+      <q-tooltip v-if="checkChanges"> Save your changes </q-tooltip>
+      <q-badge
+        v-if="checkChanges"
+        color="red"
+        rounded
+        floating
+        style="padding: 6px; border-radius: 6px"
+      />
+    </q-btn>
+    <q-btn label="Restart server" color="primary" @click="restartServer">
+      <q-tooltip v-if="needsRestart">
+        Restart the server for changes to take effect
+      </q-tooltip>
+      <q-badge
+        v-if="needsRestart"
+        color="red"
+        rounded
+        floating
+        style="padding: 6px; border-radius: 6px"
+      />
+    </q-btn>
+    <q-btn label="Topup" color="primary" @click="topUpDialog.show = true">
+      <q-tooltip> Add funds to a wallet. </q-tooltip>
+    </q-btn>
+    <!-- <q-btn
+      label="Download Database Backup"
+      flat
+      @click="downloadBackup"
+      ></q-btn> -->
+    <q-btn
+      flat
+      label="Reset to defaults"
+      color="primary"
+      @click="deleteSettings"
+      class="float-right"
+    >
+      <q-tooltip> Delete all settings and reset to defaults. </q-tooltip>
+    </q-btn>
+  </div>
+</div>
+<div class="row q-col-gutter-md justify-center">
+  <div class="col q-gutter-y-md">
+    <q-card>
+      <div class="q-pa-md">
+        <div class="q-gutter-y-md">
+          <q-tabs v-model="tab" active-color="primary" align="justify">
+            <q-tab
+              name="funding"
+              label="Funding"
+              @update="val => tab = val.name"
+            ></q-tab>
+            <q-tab
+              name="users"
+              label="Users"
+              @update="val => tab = val.name"
+            ></q-tab>
+            <q-tab
+              name="server"
+              label="Server"
+              @update="val => tab = val.name"
+            ></q-tab>
+            <q-tab
+              name="theme"
+              label="Theme"
+              @update="val => tab = val.name"
+            ></q-tab>
+          </q-tabs>
+        </div>
+      </div>
+      <q-form name="settings_form" id="settings_form">
+        <q-tab-panels v-model="tab" animated>
+          {% include "admin/_tab_funding.html" %} {% include
+          "admin/_tab_users.html" %} {% include "admin/_tab_server.html" %} {%
+          include "admin/_tab_theme.html" %}
+        </q-tab-panels>
+      </q-form>
+    </q-card>
+  </div>
+</div>
+<q-dialog v-model="topUpDialog.show" position="top">
```

Comment:
- dialog not closed after `TOPUP` clked
- the amount is not checked, the admin can `TOPUP` as much as it likes
- this  can be an issue for the `saas` model